adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapter.java

```diff
@@ -331,6 +332,7 @@ protected Router createRouter() {
      * information before the potentially expensive credentials validation is done
      * <p>
      * The default implementation updates the trace sampling priority in the execution context tracing span.
+     * It also verifies that the adapter is enabled for the tenant, failing the returned future if that is not the case.
```

Comment:
In addition to the adapter, `isAdapterEnabled(...)` also verifies if the tenant is _enabled_ or not. How about including this info about tenant also in the JavaDoc ?  It is also applicable for the MQTT adpter too. WDYT?

Reply:
I've adapted the javadocs.